lib/runner/extensions/http-request.command.js

```diff
@@ -144,7 +144,11 @@ module.exports = {
                     // Since we encountered an error before even attempting to send the request, we bubble it up
                     // here.
                     afterRequest(err, undefined, item.request);
-                    return safeNext(err, {request: item.request}, err);
+                    return safeNext(
+                        err,
```

Comment:
Why is `err` being passed twice?

Reply:
Good catch, we had to pass `err` twice to `next`. Now that is handled by `safeNext`.